gateway-js/src/index.ts

```diff
@@ -236,10 +237,23 @@ export class ApolloGateway implements GraphQLService {
     // 3. If config is `undefined`, use the default uplink URL
     if (isManagedConfig(this.config)) {
       const envEndpoint = process.env.APOLLO_SCHEMA_CONFIG_DELIVERY_ENDPOINT;
-      this.schemaConfigDeliveryEndpoint =
-        this.config.schemaConfigDeliveryEndpoint ??
-        envEndpoint ??
-        'https://uplink.api.apollographql.com/';
+      var endPoints: string[] | null = envEndpoint ? envEndpoint.split(",") : null;
```

Comment:
* `var` is perfectly valid but unconventional at this point (except in pretty specific cases).
* We can use optional chaining `?.` here in combination with nullish coalesce `??` for the same effect. What you have is also valid, just a bit less "modern".

Suggestion offered is mostly preference:
```suggestion
      const endPoints = envEndpoint?.split(",") ?? null;
```